tensorflow/python/framework/config.py

```diff
@@ -18,10 +18,22 @@
 from __future__ import division
 from __future__ import print_function
 
+from tensorflow.python import _pywrap_tf32_execution
 from tensorflow.python.eager import context
 from tensorflow.python.util import deprecation
 from tensorflow.python.util.tf_export import tf_export
 
+@tf_export('config.tensor_float_32_execution_allowed')
```

Comment:
Let's not expose this yet in TF 2.3, since TF 2.3 won't have CUDA 11 support. See [the RFC](https://github.com/tensorflow/community/pull/247) for details.

Reply:
Done